apps/backend/api/graphql/schema.graphql

```diff
@@ -2482,6 +2492,8 @@ type TopicFollowSettings {
 
 type Track {
   id: ID
+  # Whether this track requires purchased access (paid content)
+  accessControlled: Boolean
```

Comment:
seems like we should call this the same thing we call it on group, so either both are paywall (or paywalled?) or both are accessControlled. i lean towards paywalled